lms/static/scripts/frontend_apps/components/test/ErrorDialogApp-test.js

```diff
@@ -0,0 +1,50 @@
+import { mount } from 'enzyme';
+import { createElement } from 'preact';
+
+import { Config } from '../../config';
+
+import ErrorDialogApp from '../ErrorDialogApp';
+
+describe('ErrorDialogApp', () => {
+  let fakeConfig;
+
+  const renderApp = () => {
+    const config = {
+      ErrorDialog: fakeConfig,
+    };
+
+    return mount(
+      <Config.Provider value={config}>
+        <ErrorDialogApp />
+      </Config.Provider>
+    );
+  };
+
+  beforeEach(() => {
+    fakeConfig = {
+      errorCode: null,
+      errorDetails: '',
+    };
+
+    sinon.stub(window, 'close');
```

Comment:
Stubbing `window.close()` isn't needed in this test. It was needed for `OAuth2RedirectErrorApp` because that is used in a popup window and it has a button that calls `window.close()`.